src/organisms/tables/ComparisonTable/comparison_table.module.scss

```diff
@@ -0,0 +1,63 @@
+@import 'src/assets/stylesheets/global';
+
+.comparisonTable {
+  border: 1px solid $neutral-5;
+  border-radius: 6px;
+  padding: 0;
+
+  > div:last-child {
+    > div {
+      border-bottom: none;
+    }
+  }
+
+  @media #{$small-only} {
+    border: none;
+  }
+}
+
+.header {
+  padding: 24px 30px;
+  border-right: 1px solid $neutral-5;
+  border-bottom: 1px solid $neutral-5;
+
+  &:last-child {
+    border-right: none;
+  }
+
+  @media #{$small-only} {
+    border: none;
+
+    &:not( :first-child ) {
+      display: none;
+    }
+  }
+}
+
+.row {
+  padding: 18px 30px;
+  position: relative;
+  border-right: 1px solid $neutral-5;
+  border-bottom: 1px solid $neutral-5;
+
+  &:last-child {
+    border-right: none;
+  }
+
+  @media #{$small-only} {
```

Comment:
@cisacke Our current convention is to consolidate all of the @medias to the bottom of the file.